bc_obps/service/tests/test_facility_designated_operations_timeline_service.py

```diff
@@ -0,0 +1,94 @@
+from registration.schema.v1.facility_designated_operation_timeline import (
+    FacilityDesignatedOperationTimelineFilterSchema,
+)
+from service.facility_designated_operation_timeline_service import FacilityDesignatedOperationTimelineService
+import pytest
+from model_bakery import baker
+
+pytestmark = pytest.mark.django_db
+
+
+class TestGetTimeline:
+    @staticmethod
+    def test_get_timeline_by_operation_id_for_irc_user():
+        irc_user = baker.make_recipe('utils.irc_user')
+
+        timeline_for_selected_operation = baker.make_recipe(
+            'utils.facility_designated_operation_timeline', _quantity=10
+        )
+        # timeline for random operation's facilities
+        baker.make_recipe('utils.facility_designated_operation_timeline', _quantity=10)
+
+        expected_facilities = FacilityDesignatedOperationTimelineService.get_timeline_by_operation_id(
+            irc_user, timeline_for_selected_operation[0].operation.id
+        )
+
+        assert expected_facilities.count() == 10
+
+    @staticmethod
+    def test_get_timeline_by_operation_id_for_unapproved_user_industry_user():
+        industry_user = baker.make_recipe('utils.industry_operator_user')
+        facility_designated_operation_timeline = baker.make_recipe('utils.facility_designated_operation_timeline')
+        with pytest.raises(Exception):
```

Comment:
nit: A good practice would be to add the exception error as the second argument of the `.raises` method, like this: `pytest.raises(Exception, match="<error message>")`